docs/groovy-script/mods/selectionguicrafting/sgc_recipe.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Crafting Recipe"
+titleTemplate: "Selection GUI Crafting | CleanroomMC"
+description: "Creates a recipe that is shown in the specified category. It can requires an input, output and tool. You may also need a catalyst to craft the recipe."
+source_code_link: "https://github.com/Ender-Development/selection-gui-crafting-continued/blob/master/src/main/java/io/enderdev/selectionguicrafting/integration/groovyscript/SgcRecipe.java"
+---
+
+# Crafting Recipe (Selection GUI Crafting)
+
+## Description
+
+Creates a recipe that is shown in the specified category. It can requires an input, output and tool. You may also need a catalyst to craft the recipe.
```

Comment:
add a serial comma: `input, output, and tool`. 
`It can requires` should either be `It requires` or `It can require`.